packages/components/src/Paper.purs

```diff
@@ -11,6 +11,9 @@ data Paper = HBL | ON | VN | KSF | JUNIOR
 derive instance eqPaper :: Eq Paper
 derive instance ordPaper :: Ord Paper
 
+instance showPaper :: Show Paper where
```

Comment:
Not a good idea. `Paper` has multiple possible stringifications, best call what we want each time. Is there some other `Show` instance that requires `Paper` to have one too?

Reply:
Oops, missed this part of the diff when reading. I wish Purescript (and Haskell) had different type classes for 'Show for user' and 'Debug print'.

Reply:
I think `Show` in PureScript is a bit superfluous since it's always possible to view the underlying JavaScript representation of a value.